system-test/storage.ts

```diff
@@ -3277,6 +3277,19 @@ describe('storage', () => {
       assert.strictEqual(body, localFile.toString());
     });
 
+    it('should create a signed read url with usableFrom', async () => {
```

Comment:
Pending reply from the Storage backend team. I'll get back to you as soon as I get an update.